api/descriptions/policies/vulnerability_ci_images_put.md

```diff
@@ -0,0 +1,119 @@
+Adds a vulnerability rule to raise alerts or block deployments when images with vulnerabilities are detected.
```

Comment:
@KICKdesigns All of the `PUT /api/v1/policies/*` endpoints work in sort of the same way. To update the policy, you need to get the whole policy (all the rules), save it somewhere, manipulate the JSON to insert your new rule (rule ordering is important), and the PUT the whole thing back.

It's well documented in this endpoint: https://cdn.twistlock.com/docs/api/twistlock_api.html#policies_vulnerability_images_put

Given that the process, in general, is the same for all `PUT policies` endpoints, maybe we should describe the strategy in `api/descriptions/policies/policies.md`? WDYT?

Reply:
@iansk Sure, we can put a general descriptions in the main policies.md. Then perhaps (in case people don't read the main overview section, we can put a small note referencing that section in that general area?

Reply:
@KICKdesigns Sounds good

Reply:
Will submit the policies.md in a separate PR, so this branch stays "clean".